chsdi/__init__.py

```diff
@@ -58,7 +48,20 @@ def main(global_config, **settings):
     config = Configurator(settings=settings, request_factory=WsgiSchemeAdaptedRequest)
     config.include('pyramid_mako')
     config.include('akhet.static')
+
     # wrapper around all views
+    # This is a wrapper function around all views. If OPTIONS is given, an empty string will be returned
+    # As HTTP OPTIONS is not cached, this wrapper will safe some power
+    def options_view(view, info):
+        def wrapper_view(context, request):
+            if request.method == 'OPTIONS':
+                return Response()
+            # HTTP 405: method not allowed
+            elif request.method not in request_method:
+                context.code = context.status_code = 405
+            return view(context, request)
+        return wrapper_view
+
```

Comment:
Because it is not only for options I would rename it, maybe something like `def methods_view`